pkg/reconcilermanager/constants.go

```diff
@@ -141,14 +141,15 @@ const (
 	//HelmDeployNamespace is the OS env variable key for the Helm deploy namespace.
 	HelmDeployNamespace = "HELM_DEPLOY_NAMESPACE"
 
-	// HelmValues is the OS env variable key for the Helm chart values.
+	// HelmValues is the OS env variable key for the inline Helm chart values.
 	HelmValues = "HELM_VALUES"
 
-	// HelmConfigMapRef is the path where valuesFiles from ConfigMaps are mounted.
-	HelmConfigMapRef = "HELM_CONFIGMAP_REF"
+	// HelmValuesFilePath is the path where valuesFiles from a ConfigMap are mounted.
```

Comment:
based on mountConfigMapValuesFiles it looks like this isn't a full path but a path fragment? Can you explain?

Reply:
do we need to have HelmValuesFilePath and HelmValuesFilePaths be separate inputs to the reconciler? Can't we just have the reconciler-manager make a single list of paths that includes the inline values as a file?

Reply:
SG, I'll try to make it clearer what's happening with comments